pkg/kubectl/cmd/config/config.go

```diff
@@ -225,6 +225,9 @@ func ModifyConfig(configAccess ConfigAccess, newConfig clientcmdapi.Config, rela
 
 			configToWrite := getConfigFromFileOrDie(destinationFile)
 			t := *cluster
+			if configToWrite.Clusters == nil {
```

Comment:
> This code probably should move into getConfigFromFileOrDie

`NewConfig` already tries: https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/clientcmd/api/types.go#L108 .  Its call in the `Load` path and the `getConfigFromFileOrDie` path.  Is the `Decode` (https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/clientcmd/loader.go#L255) setting them back to nil?  I don't see much value in getting a nil map versus an empty map, so I'm in favor of fixing `Load`